users.tf

```diff
@@ -37,3 +37,28 @@ resource "aws_iam_group_membership" "admin_group_membership" {
   group = "${aws_iam_group.admin_group.name}"
   users = ["${aws_iam_user.admin_user.name}"]
 }
+
+# Create a platform-services user
+resource "aws_iam_user" "platform_services_user" {
+  name = "acp-platform-services-${var.environment}"
+}
+
+resource "aws_iam_group" "platform_services_group" {
+  name = "acp-platform-services-${var.environment}"
+}
+
+resource "aws_iam_policy" "platform_services_policy" {
+  name   = "acp-platform-services-${var.environment}}"
+  policy = "${file("platform_services.json")}"
```

Comment:
can we use the terraform [native](https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html) implementation .. The reason being that terraform can calculate the diff, while content can only be applied